recipe/meta.yaml

```diff
@@ -8,10 +8,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://fossies.org/linux/misc/poppler-22.12.0.tar.xz
-  # we use different source as our firewall does strange things
-  # url: https://poppler.freedesktop.org/poppler-{{ version }}.tar.xz
-  sha256: d9aa9cacdfbd0f8e98fc2b3bb008e645597ed480685757c3e7bc74b4278d15c0
+  url: https://gitlab.freedesktop.org/poppler/poppler/-/archive/poppler-{{ version }}/poppler-poppler-{{ version }}.tar.bz2
```

Comment:
This url is also available https://poppler.freedesktop.org/poppler-22.11.0.tar.xz from the home site https://poppler.freedesktop.org/releases.html